buildozer/__init__.py

```diff
@@ -282,6 +296,7 @@ def cmd(self, command, **kwargs):
             else:
                 self.debug('Run {0!r} ...'.format(command.split()[0]))
         self.debug('Cwd {}'.format(kwargs.get('cwd')))
+        self.log_env(self.DEBUG, kwargs["env"])
```

Comment:
I think that like should be removed.
I'm seeing it only now after using buildozer master for a while.
For instance during the auto accept license process, this `log_env` seems to be printing every time. We want it only after crashes, not for every command call

Reply:
I've created a PR to remove that line https://github.com/kivy/buildozer/pull/843